dbt/models/default/schema/default.vw_pin_condo_char.yml

```diff
@@ -104,7 +104,7 @@ models:
             (is_common_area OR is_parking_space)
             AND (char_bedrooms IS NOT NULL OR char_full_baths IS NOT NULL)
           config:
-            error_if: ">48205" # as of 2024-06-03
+            error_if: ">48206" # as of 2024-06-14
```

Comment:
This feels like it's going to be an ongoing issue, now. I'll take another look next week...

Reply:
See #515.